README.md

```diff
@@ -4,43 +4,68 @@
 [![Build Status](https://travis-ci.org/Azure/azure-sdk-for-go.svg?branch=master)](https://travis-ci.org/Azure/azure-sdk-for-go) 
 [![Go Report Card](https://goreportcard.com/badge/github.com/Azure/azure-sdk-for-go)](https://goreportcard.com/report/github.com/Azure/azure-sdk-for-go)
 
-azure-sdk-for-go provides Go packages for using Azure services. It has been
-tested with Go 1.8 and 1.9. To be notified about updates and changes, subscribe
-to the [Azure update feed][].
+azure-sdk-for-go provides Go packages for managing and using Azure services. It has been
+tested with Go 1.8, 1.9 and 1.10.
```

Comment:
I think it would be good to reword this so that we don't have to update it every time a new version of Go is released.  Maybe something along the lines of the latest version plus the previous two?

Reply:
agree, but can we fix this later?